src/edu/csus/ecs/pc2/ui/SubmitSampleRunsPane.java

```diff
@@ -0,0 +1,877 @@
+		bottomPane.add(submitRunButton);
+
+//		centerPane2.add(controlsPane, BorderLayout.CENTER);
+		JScrollPane leftSide = new JScrollPane(controlsPane);
+		
+		JPanel submitNoSamplesPane = new JPanel();
+		submitNoSamplesPane.setBorder(new TitledBorder(null, "", TitledBorder.LEADING, TitledBorder.TOP, null, null));
+		FlowLayout fl_submitNoSamplesPane = (FlowLayout) submitNoSamplesPane.getLayout();
+		fl_submitNoSamplesPane.setAlignment(FlowLayout.LEFT);
+		submitNoSamplesPane.setBounds(33, 131, 650, 32);
+		controlsPane.add(submitNoSamplesPane);
+
+		checkBoxSubmitFailingSamples = new JCheckBox("Submit Failing Samples");
+		checkBoxSubmitFailingSamples.setVerticalAlignment(SwingConstants.TOP);
+		submitNoSamplesPane.add(checkBoxSubmitFailingSamples);
+		checkBoxSubmitFailingSamples.setSelected(true);
+		checkBoxSubmitFailingSamples.setToolTipText("Submt all non-AC (Yes) submissions");
+
+		judgingTypesButtonGroup.add(submitAllJudgingTypesRadioButton);
+		submitAllJudgingTypesRadioButton.setSelected(true);
+		judgingTypesButtonGroup.add(submitSelectedJudgingTypeRadioButton);
+		
+		submitNoSamplesPane.add(submitAllJudgingTypesRadioButton);
+		submitNoSamplesPane.add(submitSelectedJudgingTypeRadioButton);
+		
+		leftSide.setPreferredSize(new Dimension(230, 230));
+		leftSide.setVerticalScrollBarPolicy(ScrollPaneConstants.VERTICAL_SCROLLBAR_AS_NEEDED);
+		leftSide.setWheelScrollingEnabled(true);
+		splitPane.setLeftComponent(leftSide);
+
+		// centerPane2.add(LogPanel, BorderLayout.SOUTH);
+		JScrollPane logScrollPanel = new JScrollPane(LogPanel);
+		logScrollPanel.setPreferredSize(new Dimension(60, 60));
+		logScrollPanel.setVerticalScrollBarPolicy(ScrollPaneConstants.VERTICAL_SCROLLBAR_AS_NEEDED);
+		logScrollPanel.setWheelScrollingEnabled(true);
+		splitPane.setRightComponent(logScrollPanel);
+		
+		
+		selectedNosLabel.setToolTipText("None selected");
+		selectedNosLabel.addMouseListener(new MouseAdapter() {
+			@Override
+			public void mouseClicked(MouseEvent e) {
+			    showJudgementTypeList();
+			}
+		});
+		submitNoSamplesPane.add(selectedNosLabel);
+		JButton selectJudementTypesButton = new JButton("Select");
+		selectJudementTypesButton.setToolTipText("Select Judgement Types");
+		selectJudementTypesButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+			    showJudgementTypeList();
+			}
+		});
+		
+		submitNoSamplesPane.add(selectJudementTypesButton);
+		
+		JLabel lblNewLabel_2 = getWhatsThisOne();
+		submitNoSamplesPane.add(lblNewLabel_2);
+		
+		JPanel submitNoSamplesPane_1 = new JPanel();
+		FlowLayout flowLayout_2 = (FlowLayout) submitNoSamplesPane_1.getLayout();
+		flowLayout_2.setAlignment(FlowLayout.LEFT);
+		submitNoSamplesPane_1.setBorder(new TitledBorder(null, "", TitledBorder.LEADING, TitledBorder.TOP, null, null));
+		submitNoSamplesPane_1.setBounds(33, 161, 650, 32);
+		controlsPane.add(submitNoSamplesPane_1);
+		   
+        checkBoxProblems = new JCheckBox("Problems");
```

Comment:
As noted above, unexplained changes in indentation (such as on the above line) are confusing.  It would be much nicer if a  Ctrl-Shift-F was applied to the code.

Even better would be if the constructor for this class constructor didn't contain over 250 lines of code but instead called methods to construct each of the various GUI components; it would be a lot easier to follow.

Reply:
Regarding the inconsistent indentation, see my comment above:  when I pull the code into Eclipse, I no longer see the weird indentation inconsistencies.  Maybe a GitHub anomaly?

In any case, I still think the constructor shouldn't consist of over 250 lines of code (it should be broken down into method calls).